### DISCUSSION TITLE
Launch with an empty state?

### DISCUSSION THREAD

**Discussion by failable** [zed-industries/zed#6519] [Feature Requests]
After I close all workspace windows and quit, the next time I launch Zed, it will still open the last closed workspace. How can I start with an empty state?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
